### PR TITLE
feat: create constants folder

### PR DESCRIPTION
**Pull request by mthanh2209**

### CODE REVIEW

File: javascript-practice/src/javascripts/constants/messages.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/**
+ * Message for alert the error while fetching or rendering.
+ */
+export const ERROR_MESSAGE = {
+	GET_PPRODUCT_LIST: "Server error! Couldn't load product list",
+	GET_CATEGORY_LIST: "Server error! Couldn't load category list",
+
+	ADD_CART: "Couldn't add to cart",
+
```
Comment (hungnguyenhuu-agilityio) [inline javascript-practice/src/javascripts/constants/messages.js:9]: Why we have these break-lines?

File: javascript-practice/src/javascripts/constants/messages.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+/**
+ * Message for alert the error while fetching or rendering.
+ */
+export const ERROR_MESSAGE = {
+	GET_PPRODUCT_LIST: "Server error! Couldn't load product list",
+	GET_CATEGORY_LIST: "Server error! Couldn't load category list",
+	ADD_CART: "Couldn't add to cart",
+	SERVER_ERROR: "Server Error! Please try again later",
```
Comment (hungnguyenhuu-agilityio) [inline javascript-practice/src/javascripts/constants/messages.js:8]: The message of line 5, 6, 8 look the same? Should we split them like that? Do you have specific case for each error?

Comment (mthanh2209) [reply javascript-practice/src/javascripts/constants/messages.js:7]: I fixed it. Please help me check again

Comment (hungnguyenhuu-agilityio) [reply javascript-practice/src/javascripts/constants/messages.js:7]: Update the naming for line 5-6

Comment (hungnguyenhuu-agilityio) [reply javascript-practice/src/javascripts/constants/messages.js:7]: Seems like you missed this message

Comment (hungnguyenhuu-agilityio) [reply javascript-practice/src/javascripts/constants/messages.js:7]: The constant name should show the idea of the message. @mthanh2209

**Pull request merged**